_maps/snake.json

```diff
@@ -0,0 +1,28 @@
+{
+    "map_name": "NSV Snake",
+    "map_path": "map_files/Snake",
+    "map_file": [
+		"snake_lower.dmm", 
+		"snake_upper.dmm"
+	], 
+    "traits": [
+		{
+			"Up": 1,
+			"linkage": "self"
+		}, 
+		{
+			"Down": 1,
```

Comment:
```suggestion
			"Down": -1,
```

:)

Reply:
This apparently didn't fix the linkage issue.